docs/yaml/generate.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
Could this be moved to `scripts/docs/generate-yaml.sh` to be consistent with our other scripts?